tutorials/cloudsdk-js-generator/cloudsdk-js-generator.md

```diff
@@ -13,6 +13,7 @@ primary_tag: products>sap-cloud-sdk
 > This tutorial is not actively maintained and might be partially outdated.
 > Always up-to-date documentation is published on our [documentation portal](https://sap.github.io/cloud-sdk/).
 > We will provide a link to the updated version of this tutorial as soon as we release it.
+> In this tutorial, we use version 1 of the SAP Cloud SDK for TypeScript/JavaScript.
```

Comment:
Please don't use the "royal we".
Better say something like: "In this tutorial version 1 of the SAP Cloud SDK for TypeScript/JavaScript is used."